src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -265,8 +265,9 @@ rel="preload">` has been fired.
 ```
 
 Warning: For cross-origin resources, make sure your CORS headers are set
-properly because opaque responses retrieved with `fetch(videoFileUrl, { mode:
-'no-cors' })` are not allowed to feed any video or audio element.
+properly. As we can't create an array buffer from an opaque response retrieved
```

Comment:
There's still a second occurrence (later in the MSE portion of this doc) that is less explanatory (it doesn't mention array buffer, for example). Make the two warnings the same?